metricbeat/module/openmetrics/_meta/fields.yml

```
@@ -7,18 +7,41 @@
   fields:
     - name: openmetrics
       type: group
+      #      release: beta/ga
```

:slightly_smiling_face: 
```suggestion
      release: beta
```

---

Changed to beta, uncommented.